### PR TITLE
create review manager

### PR DESCRIPTION
**Pull request by maciej-MKan**
This is my first piece of code. I wrote a simple review manager to check the ability to navigate the project. So.. don't kill me.

### CODE REVIEW

File: backend/apps/books/managers.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class ReviewManager(models.Manager):
+    def all_reviews(self):
```
Comment (Indoxer) [inline backend/apps/books/managers.py:5]:
```
from backend.apps.books.models import Review

# it breaks DRY and it is confusing. You have the same thing 2 times.
Review.objects.all_reviews() 
Review.objects.all() # it also works
```

**Review by Indoxer — CHANGES_REQUESTED** [review]

File: backend/apps/books/managers.py (hunk 1 above)
Comment (maciej-MKan) [reply backend/apps/books/managers.py]: I removed the all_reviews method for not duplicate functionality.

**Review by kwiats — COMMENTED** [review]
ok

**Review by kwiats — APPROVED** [review]
lgfm

**Pull request merged**